angular-techDraft/src/app/my-nav/my-nav.component.html

```diff
@@ -0,0 +1,13 @@
+<div class="ui fluid placeholder">
+  <div class="image header">
+    <div class="container">
+      <img class="ui mini circular image" src="https://yt3.ggpht.com/-5oE3xInEpSw/AAAAAAAAAAI/AAAAAAAAAAU/5Q6ic5I5378/s88-mo-c-c0xffffffff-rj-k-no/photo.jpg">
+    </div>
+    <div><i class="cog icon"></i></div>
+  </div>
+  <div class="paragraph">
+    <div class="line">
+      <h2>Your Profile </h2>
```

Comment:
Spacing is required in between the picture placeholder and the <h2> tag of Your Profile.